File: playerFeature/src/main/java/com/yes/player/presentation/FontView.java
package com.yes.player.presentation;

import android.content.Context;
import android.graphics.Typeface;
import android.util.AttributeSet;

public class FontView extends androidx.appcompat.widget.AppCompatTextView{
    public FontView(Context context)
    {
        super(context);
        init(context);
    }

    public FontView(Context context, AttributeSet attrs)
    {
        super(context, attrs);
        init(context);
    }

    public FontView(Context context, AttributeSet attrs, int defStyleAttr)
    {
        super(context, attrs, defStyleAttr);
        init(context);
    }
    private void init(Context context){
        Typeface tf = Typeface.createFromAsset(context.getAssets(), "a_lcdnova_allfont.ttf");
        this.setTypeface(tf);
    }
}

File: playerFeature/src/main/java/com/yes/player/presentation/ui/FontView.kt
package com.yes.player.presentation.ui

import android.content.Context
import android.graphics.Typeface
import android.util.AttributeSet
import androidx.appcompat.widget.AppCompatTextView

class FontView : AppCompatTextView {
    constructor(context: Context) : super(context) {
        init(context)
    }

    constructor(context: Context, attrs: AttributeSet?) : super(context, attrs) {
        init(context)
    }

    constructor(context: Context, attrs: AttributeSet?, defStyleAttr: Int) : super(
        context,
        attrs,
        defStyleAttr
    ) {
        init(context)
    }

    private fun init(context: Context) {
        val tf = Typeface.createFromAsset(context.assets, "a_lcdnova_allfont.ttf")
        this.typeface = tf
    }
}